uaclient/tests/test_contract.py

```diff
@@ -21,9 +21,9 @@ class TestProcessEntitlementDeltas:
     def test_error_on_missing_entitlement_type(self):
         """Raise an error when neither dict contains entitlement type."""
         new_access = {'entitlement': {'something': 'non-empty'}}
-        error_msg = 'Could not determine contract delta service type %s %s' % (
-            {},
-            new_access,
+        error_msg = (
+            'Could not determine contract delta service type'
+            ' {} {}'.format({}, new_access)
```

Comment:
This just looks a bit funky. How about   
error_msg = 'Could not determine contract delta service type {{}} {}'.format(new_access)  ?

Reply:
Done.